Snabble/UI/DynamicStack/DynamicStackView.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  DynamicStackView.swift
+//  Snabble
+//
+//  Created by Andreas Osberghaus on 31.08.22.
+//
+
+import Foundation
+import SwiftUI
+
+public struct DynamicStackView: View {
+    @ObservedObject public var viewModel: DynamicStackViewModel
+
+    init(viewModel: DynamicStackViewModel) {
+        self.viewModel = viewModel
+    }
+
+    public var body: some View {
+        ScrollView {
+            VStack(alignment: .center) {
+                ForEach(viewModel.widgets, id: \.id) { widget in
+                    switch widget.type {
+                    case .text:
+                        EmptyView()
+                    case .image:
+                        EmptyView()
+                    case .button:
+                        EmptyView()
+                    case .information:
+                        EmptyView()
+                    case .purchases:
+                        EmptyView()
+                    }
+                }
+            }
```

Comment:
Es ist ein VStack anstatt eine List, da es aktuell so implementiert ist und die Widgets ihre States behalten sollten. Falls wir in Probleme rennen das es zu viele werden sollten wir es in eine List umbauen.